bitrise.yml

```diff
@@ -1,10 +1,8 @@
-format_version: "2"
+format_version: "9"
```

Comment:
Currently, you are running the audit twice here:
https://github.com/bitrise-steplib/steps-certificate-and-profile-installer/pull/54/files#diff-801b81b028645c52cf0443365fea2f93L18-L29

You could remove the script step since the `audit-this-step` wf will care about it.